client/my-sites/email/email-providers-stacked-comparison/index.tsx

```diff
@@ -50,6 +52,27 @@ type EmailProvidersStackedComparisonProps = {
 	gSuiteAnnualProduct?: any;
 };
 
+// eslint-disable-next-line @typescript-eslint/no-empty-function
+const noop = () => {};
+
+const ProfessionalEmailComparisonObject: ProviderComparison = {
+	header: <h1> Professional Email </h1>,
+	tools: translate( 'Integrated email management, Inbox, calendar and contacts' ),
+	storage: translate( '30 GB storage' ),
```

Comment:
We have used `30GB storage` in the past. Is this a deliberate change?

Reply:
No, I just followed mock ups, I will reuse `30GB storage` then.